homeassistant/components/velbus/config_flow.py

```diff
@@ -82,3 +85,40 @@ async def async_step_user(
             ),
             errors=self._errors,
         )
+
+    async def async_step_usb(self, discovery_info: usb.UsbServiceInfo) -> FlowResult:
+        """Handle USB Discovery."""
+        vid = discovery_info.vid
+        pid = discovery_info.pid
+        serial_number = discovery_info.serial_number
+        device = discovery_info.device
+        manufacturer = discovery_info.manufacturer
+        description = discovery_info.description
+        await self.async_set_unique_id(
+            f"{vid}:{pid}_{serial_number}_{manufacturer}_{description}"
+        )
```

Comment:
Is it compulsory to use all 6 keys for the unique id? Is the serial number not unique enough?